eng/pipelines/aggregate-reports.yml

```diff
@@ -22,25 +30,63 @@ jobs:
           artifactName: packages
           targetPath: '$(System.ArtifactsDirectory)/BuildArtifacts'
 
-      - pwsh: Remove-Item Microsoft.Azure.*
-        workingDirectory: $(System.ArtifactsDirectory)/BuildArtifacts
-        displayName: Remove Track 1 Packages
+      - pwsh: |
+          mkdir "$(System.ArtifactsDirectory)/Packages"
```

Comment:
I recommend single quotes for paths you don't want processed (see comment above).

Reply:
Same below and everywhere.